api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/PaymentCycleVoucherEntitlementTestDataFactory.java

```diff
@@ -42,6 +43,20 @@ public static PaymentCycleVoucherEntitlement aPaymentCycleVoucherEntitlement(Loc
         );
     }
 
+    public static PaymentCycleVoucherEntitlement aPaymentCycleVoucherEntitlementWithBackdatedVouchersForYoungestChild(
+            LocalDate startDate, List<LocalDate> childrensDob) {
+        LocalDate birthdate = childrensDob.stream().max(LocalDate::compareTo).get();
```

Comment:
Why is this the eldest child's date of birth rather than the youngest?

Reply:
oops. Good spot :) Shows I have a flaw in the test